Lib/asyncio/format_helpers.py

```diff
@@ -19,19 +19,23 @@ def _get_function_source(func):
     return None
 
 
-def _format_callback_source(func, args):
-    func_repr = _format_callback(func, args, None)
+def _format_callback_source(func, args, *, debug=False):
+    func_repr = _format_callback(func, args, None, debug=debug)
     source = _get_function_source(func)
     if source:
         func_repr += f' at {source[0]}:{source[1]}'
     return func_repr
 
 
-def _format_args_and_kwargs(args, kwargs):
+def _format_args_and_kwargs(args, kwargs, *, debug=False):
     """Format function arguments and keyword arguments.
 
     Special case for a single parameter: ('hello',) is formatted as ('hello').
     """
+    # Avoid printing sensitive argument contents unless debugging
+    if not debug:
+        return '()'
```

Comment:
Perhaps `'(...)'`?

Reply:
I figured `'()'` was more in line with how existing strings chose to omit arguments. E.g. how `base_futures.py` does it, or `pwd.getpwnam()`.

But I can live with `'(...)'` if you feel strongly about it.

Reply:
Okay, that's fair.